Src/Support/Google.Apis.Auth/OAuth2/ExternalAccountCredential.cs

```diff
@@ -120,6 +133,18 @@ internal Initializer(ExternalAccountCredential other) : base(other)
         /// </summary>
         public string ServiceAccountImpersonationUrl { get; }
 
+        /// <summary>
+        /// The GCP project ID or project number to be used for Workforce Pools
```

Comment:
At the time I wrote the doc the project ID was not yet supported. Not sure if that changed, just FYI

Reply:
[Public docs](https://cloud.google.com/iam/docs/workforce-obtaining-short-lived-credentials#generate_a_configuration_file) still only mention project number, I'll change the text to only mention project number.